src/lib/accounts.js

```diff
@@ -38,3 +38,18 @@ export function getAccountsByType (cozy, accountType, accountsIndex) {
     })
   }
 }
+
+export function getAllAccounts (cozy, accountsIndex) {
+  if (accountsIndex) {
```

Comment:
It's the case, but I preferred to store the index to limit useless server requests here. Event if it's idempotent, we have to fetch the index ID and check for this index each time we want to list accounts here. What do you think? I know that worked like this in cozy-photos.